### DISCUSSION TITLE
Implement custom layer API for Qt platform (and QML plugin)

### DISCUSSION THREAD

**Discussion by ntadej** [maplibre/maplibre#173] [Native feature proposals]
Currently Qt bindings are very basic. Most of work was historically invested into QML QtLocation plugin. Now generic plugin parameters will no longer be supported so it will become impossible to render custom layers.

A basic API should be defined for C++ and QML-based MapLibre Native bindings that allows basic customisability of the map.

**Comment by wipfli** [comment]
Thanks for this suggestion. I think it would be best to open an issue in MapLibre GL Native if there is not already one for this issue.
